subsys/caf/modules/ble_state.c

```diff
@@ -205,6 +205,9 @@ static void connected(struct bt_conn *conn, uint8_t error)
 			LOG_INF("Already bonded to %s", log_strdup(addr_str));
 			goto disconnect;
 		}
+	}
+
+	if (IS_ENABLED(CONFIG_CAF_BLE_STATE_SECURITY_REQ)) {
```

Comment:
co z rolą?

Reply:
Since HIDS GATT characteristic require encryption on access, the GATT Client would automatically request security if he wants to access HIDS. I noticed that this is handled properly by our dongles, Windows 10, Android and iOS phone.

Reply:
I will add this to release notes of CAF BLE state and nRF Desktop to ensure that customers are aware. I will also inform that there is Kconfig option that could be enabled if it's needed to workaround issues of some hosts.